docs/apigateway.md

```diff
@@ -1,53 +1,105 @@
-# API Gateway (Experimental)
+# API Gateway
 
-[Web Actions](webactions.md) are out from experimental phase.
+OpenWhisk actions can benefit from being managed by API management.
 
-Web Actions allows you to invoke an action with HTTP methods other than POST and without the action's authorization API key.
+The API Gateway acts as a proxy to [Web Actions](webactions.md) and provides them with additional features features including HTTP method routing, security and rate limiting policies and more.
 
-As a result of user feedback, Web Actions are the programming model chosen to build OpenWhisk actions capable of handling HTTP events.
+Web Actions allows you to invoke an action with HTTP methods other than POST and in a public way without the action's authorization API key.
```

Comment:
I think that statement belongs more on the web action docs. once you make an action a web action its public, and it means the burden of authentication and authorization is on you the developer. API Gateway is just a proxy here.